CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
```

Comment:
Lijkt me goed, maar ik denk dat dit wel een versietag mag krijgen als het gereleased wordt? Er zitten nog wel wijzigingen in van enige importantie.

Reply:
Jazeker. Ik heb em nu als `rc.1` getagged en wil en testen in wat projecten. Zeker omdat die `addChild` aanpassing enige tijd gelden voor problemen zorgde (toen de verkeerde aanpassing, nu de juiste hopelijk).